s3-eventbridge-glue/example-pattern.json

```diff
@@ -0,0 +1,63 @@
+{
+    "title": "Automatic Data Transformation and Catalog with Amazon S3 notifications and AWS Step functions",
```

Comment:
`title`: title does not meet maximum length of 75.

Can you shorten the title and also include Glue in there as well?

Reply:
Resolved